source/documentation/deploying_apps/deploying_rails.md

```diff
@@ -135,6 +135,26 @@ You can add another instance of your app by running:
 
 ``cf scale APPNAME -i 2``
 
+### Deploying a Rails 5.1+ app with Webpack
+
+Rails 5.1+ allows you to use [webpacker](https://github.com/rails/webpacker) to install and use node.js modules.
+
+You will need to make the following changes to successfully deploy:
+
+1. Specify multiple `buildpacks` in manifest.yml instead of just one, including both the Ruby and node.js buildpacks from GitHub:
```

Comment:
Might be worth linking to the buildpacks section of the documentation, just for further information